generators/python/src/fern_python/generators/sdk/context/sdk_generator_context.py

```diff
@@ -33,11 +33,16 @@ def __init__(
         self.generator_config = generator_config
         self.pydantic_generator_context = PydanticGeneratorContextImpl(
             ir=ir,
-            type_declaration_referencer=TypeDeclarationReferencer(skip_resources_module=custom_config.improved_imports),
+            type_declaration_referencer=TypeDeclarationReferencer(
+                skip_resources_module=custom_config.improved_imports,
+                use_typeddict_requests=custom_config.pydantic_config.use_typeddict_requests,
```

Comment:
Is it weird for the pydantic config to contain `use_typeddict_requests` -- this feels like it should be an only sdk config and at the top level

Reply:
I can probably remove it easily, it was model construction related, so I put it there originally
But not wed to that at all